libs/src/services/solana/SolanaWeb3Manager.ts

```diff
@@ -324,9 +324,9 @@ export class SolanaWeb3Manager {
    * Gets the info for a user bank/wAudio token account given a solana address.
```

Comment:
nit: let's change this comment to say SPL-Token address rather than Solana Address